auth/pat.go

```diff
@@ -742,22 +373,22 @@ type PATS interface {
 	RevokePATSecret(ctx context.Context, token, patID string) error
 
 	// AddScope function adds a new scope entry.
-	AddPATScopeEntry(ctx context.Context, token, patID string, platformEntityType PlatformEntityType, optionalDomainID string, optionalDomainEntityType DomainEntityType, operation OperationType, entityIDs ...string) (Scope, error)
+	AddScopeEntry(ctx context.Context, token, patID string, scope []Scope) (ScopesPage, error)
 
```

Comment:
Lets return only newly added scopes instead of ScopesPage
```suggestion
	AddPATScopeEntry(ctx context.Context, token, patID string, entityType EntityType, optionalDomainID string, operation Operation, entityIDs ...string) ([]Scope, error)
```